corecfg/corecfg.go

```diff
@@ -53,6 +53,9 @@ func Run(tr Conf) error {
 	if err := handleProxyStore(tr); err != nil {
 		return err
 	}
+	if err := validateRefreshSchedule(tr); err != nil {
```

Comment:
(nitpick)  should we call this "handle*" as the rest?  if we think is worth calling this "validate*" for clarity, then probably handleProxyStore should also be called validateProxyStore (it doesn't have side-effects as well)

Reply:
+1 for consistency. I slightly prefer to rename the calls that just validate so `valid{ProxyStore,RefreshSchedule}`. If that is ok with you I can do this in this PR. But no super strong opinion other than it should be both validate or both handle.

Reply:
than's fine here